apps/proxy/src/app/create-project/create-project.component.ts

```diff
@@ -0,0 +1,208 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, Validators, FormBuilder, FormArray } from '@angular/forms';
+import { CAMPAIGN_NAME_REGEX, ONLY_INTEGER_REGEX, URL_REGEX } from '@proxy/regex';
+import { CustomValidators } from '@proxy/custom-validator';
+import { environment } from '../../environments/environment';
+import { CreateProjectComponentStore } from './create-project.store';
+import { Observable, takeUntil } from 'rxjs';
+import { IPaginatedResponse } from '@proxy/models/root-models';
+import { IEnvironments, IProjects } from '@proxy/models/logs-models';
+import { BaseComponent } from '@proxy/ui/base-component';
+import { select, Store } from '@ngrx/store';
+import { IAppState, selectAllProjectList } from '../ngrx';
+import { rootActions } from '../ngrx/actions';
+
+@Component({
+    selector: 'proxy-create-project',
+    templateUrl: './create-project.component.html',
+    styleUrls: ['./create-project.component.scss'],
+    providers: [CreateProjectComponentStore],
+})
+export class CreateProjectComponent extends BaseComponent implements OnInit {
+    public primaryDetailsForm: FormGroup;
```

Comment:
Type for FormGroup is still missing 
If we assign type of variable `FormGroup` without giving it type it becomes `FormGroup<any>` type 
Assign type by passing an interface into it like `FormGroup<IPrimaryDetailsForm>`